Update remove-former-employee-step-1.md (#12886)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/add-users/remove-former-employee-step-1.md

description: "Global administrators can block a former employee from logging in and block their access to Microsoft 365 services."
---

## What you need to know about terminating an employee's email session

> Here's information about how to get an employee out of email (Exchange).

Reviewer: ```suggestion
The following information explains how to remove an employee from the organization's email (Exchange):
```

> 
> <br>
> 
> ****

Reviewer: ```suggestion
```

|What you can do|How you do it|
|:-----|:-----|

> |Terminate a session (such as Outlook on the web, Outlook, Exchange active sync, etc.) and force to open a new session|Reset password|
> |Terminate a session and block access to future sessions (for all protocols)|Disable the account. For example, in the Exchange admin center or using PowerShell: <p>  `Set-Mailbox user@contoso.com -AccountDisabled:$true`|
> |Terminate the session for a particular protocol (such as ActiveSync)|Disable the protocol. For example, in the Exchange admin center or using PowerShell: <p>  `Set-CASMailbox user@contoso.com -ActiveSyncEnabled:$false`|

Reviewer: ```suggestion
|Terminate a session (such as Outlook on the web, Outlook, or Exchange active sync) and force them to open a new session.|Reset password.|
|Terminate a session and block access to future sessions (for all protocols).|Disable the account, for example, in the Exchange admin center or using PowerShell: <p>  `Set-Mailbox user@contoso.com -AccountDisabled:$true`.|
|Terminate the session for a particular protocol (such as ActiveSync).|Disable the protocol, for example, in the Exchange admin center or using PowerShell: <p>  `Set-CASMailbox user@contoso.com -ActiveSyncEnabled:$false`.|
```

> |
> 
> The above operations can be done in three places:

Reviewer: ```suggestion

These operations can be done in three different places:
```

>   
> <br>
> 
> ****

Reviewer: ```suggestion
```

> |If you terminate the session here|How long it takes|
> |---|---|
> |In the Exchange admin center or using PowerShell|Expected delay is within 30 min|
> |In the Azure Active Directory admin center|Expected delay is 60 min|
> |In an on-premises environment|Expected delay is 3 hours or more|
> |
> 

Reviewer: ```suggestion
|If you terminate the session here|How long does it take|
|---|---|
|In the Exchange admin center or using PowerShell.|Expected delay is approximately 30 minutes.|
|In the Azure Active Directory admin center.|Expected delay is 60 minutes.|
|In an on-premises environment.|Expected delay is 3 hours or more.|
```
